sdk/storage/Azure.Storage.Blobs/src/Models/ObjectReplicationPolicy.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Contains Object Replication Policy ID and the respective list of
+    /// <see cref="ObjectReplicationRule"/>(s). This is used when retrieving the
+    /// Object Replication Properties on the source blob. The policy id for the
+    /// destination blob is set in ObjectReplicationDestinationPolicy of the respective
+    /// method responses. (e.g. <see cref="BlobProperties.ObjectReplicationDestinationPolicy"/>,
+    /// <see cref="BlobDownloadDetails.ObjectReplicationDestinationPolicy"/>).
+    /// </summary>
+    public class ObjectReplicationPolicy
+    {
+        internal ObjectReplicationPolicy() { }
```

Comment:
I think we have a SomethingFactory class where customers can create these for test purposes. We should add ORS entities there as well to factory method for the type enclosing them.

Reply:
@amnguye you can add methods to create `ObjectReplicationPolicy` and `ObjectReplicationRule` in the  `BlobsModelFactory`.